source/includes/fact-mongosh-shell-method-toc.rst

```diff
@@ -0,0 +1,18 @@
+The methods listed on this table of contents page are
+:mongosh:`mongosh </>` methods.  This is *not* the documention for
+``Node.js`` or other programming language specific driver methods.
+
+In most cases :mongosh:`mongosh </>` methods work the same way as the
+legacy :binary:`~bin.mongo` shell methods. However, some legacy methods
+are unavailable in ``mongosh``.
+
+For legacy :binary:`~bin.mongo` shell documention, refer to the
+documentation for the corresponding ``MongoDB`` release.
+
+   - :v4.4:`mongo shell v4.4 </mongo>`
```

Comment:
This bulleted list should not be indented like this. It will show an "unexpected indentation" error in the build log.